### PR TITLE
rename redhat-cop to konveyor where appropriate

### LINKED ISSUE #351: Docs and default configuration use redhat-cop URL; should use konveyor URL

### PR DESCRIPTION
**Pull request by weshayutin**
* Rename redhat-cop to konveyor where appropriate

## Describe the behavior changes introduced in this PR
none, just renaming the git url

## Linked Issues?
** https://github.com/konveyor/pelorus/issues/351 **

resolves #351 <-- Use this if merging should auto-close an issue

related to #<issue number> <-- Use this if it shouldn't

## Testing Instructions

Please include any additional commands or pointers in addition to our [standard PR testing process](/docs/Development.md#testing-pull-requests).

@redhat-cop/mdt

### CODE REVIEW

**Review by mpryc — APPROVED** [review]

**Comment by mpryc** [comment]
@weshayutin if you are doing any change to the Charts they require version bump. There is script for that.

**Review by mpryc — CHANGES_REQUESTED** [review]
As in my comment, required chart version bump.

**Comment by weshayutin** [comment]
@mpryc ok.. thanks for the review!  I'll review the developer guide and scripts :)
Thank you

File: docs/Development.md [hunk 1]
```diff
@@ -314,9 +314,9 @@ We are in the process of refactoring our helm charts such that they can be teste
 
 The following is a walkthrough of the process we follow to create and manage versioned releases of Pelorus.
 
-1. Update the default exporter tag version to the new tag you're about to make.
```
Comment (KevinMGranger) [inline docs/Development.md:317]: Two trailing spaces in markdown is significant. It puts a `<br>` instead of starting a new `<p>`. Can you put these back?

**Review by KevinMGranger — CHANGES_REQUESTED** [review]
Aside from the one newline thing, this looks good to me!

**Comment by weshayutin** [comment]
OK.. thanks for the review Kevin, Michal.
I scanned the markdown files for other places where two spaces impacted the indentation as well and I think I found them all.
bumped the chart to $next 1.4.8

thanks again!

**Review by mpryc — APPROVED** [review]

**Review by KevinMGranger — CHANGES_REQUESTED** [review]
Once the Chart.yaml conflicts are fixed, this is good to go!

**Review by mpryc — APPROVED** [review]

**Review by KevinMGranger — APPROVED** [review]

**Pull request merged**